src/ci/resources/papi_v2beta_application.conf

```diff
@@ -77,5 +77,19 @@ backend {
         include "papi_v2beta_provider_config.inc.conf"
       }
     }
+    ################################################################################
+    ######   THIS BACKEND IS EXCLUSIVE FOR PAPI V2 BETA AND THIS IS THE    #########
+    ######   REASON WHY CONFIG VALUES FOR IT ARE DEFINED HERE INSTEAD OF   #########
+    ######               papi_v2_shared_application.inc.conf               #########
+    ################################################################################
```

Comment:
```suggestion
    ######     DOCKER IMAGE CACHING IS NOT AVAILABLE ON PAPI V2 ALPHA      #########
    ################################################################################
```

Reply:
I think longer explanation is better in this case in order to make it clear why this backend defined in different fashion than others

Reply:
I agree, my suggestion is adding that line to the existing lines 🙂

Reply:
Sorry, didn't realize that this is an addition, not a replacement.